src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -1955,6 +1959,9 @@ private static bool HasSpecialIntPtrConversion(TypeSymbol source, TypeSymbol tar
             }
 
             return false;
+
+            static bool isIntPtrOrUIntPtr(TypeSymbol type) =>
```

Comment:
This check is repeated in `EmitConversions.cs`. Did you consider pulling this out into a re-usable helper? Not sure if two cases is enough.

Reply:
> This check is repeated in EmitConversions.cs. Did you consider pulling this out into a re-usable helper? 

Was the check in EmitConversions.cs used for `Debug.Assert()` only? If so, it may not be worth sharing the code since the check is simple.

---
In reply to: [489080353](https://github.com/dotnet/roslyn/pull/47708#discussion_r489080353) [](ancestors = 489080353)